tests/template_tests/syntax_tests/test_for.py

```diff
@@ -197,3 +198,37 @@ def test_for_tag_context(self):
             },
         })
         self.assertEqual(output, 'two:2,four:4,_six:6,eight:8,')
+
+    @setup({'invalid_for_loop': '{% for x items %}{{ x }}:{{ y }}{% endfor %}'})
+    def test_for_tag_invalid_arg(self):
```

Comment:
You can omit `for_tag_` in the test name. Having it on every test name in the class seems silly.